nservicebus/upgrades/5to6/raven-upgrade.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Upgrade Raven from NServiceBus 5 to 6
```

Comment:
I was wondering about that and intentionally chose NServiceBus numbers. I feel like anyone looking into this upgrade guide would only be doing this because they are jumping from 5 to 6. They in fact could not if they weren't. 

You don't think we should be uniform in the upgrade guides by using core's major?